.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
please remove this file, you can rather add `.vscode` to the main `.gitignore` file.